examples/calendar/ReadMe.md

```diff
@@ -0,0 +1,79 @@
+## Problem Statement
+
+Right now when we use calendly, it has gives access to the scheduler and what times they are not available. This might be an issue for folks are serious
+about privacy of their
```

Comment:
@vaibhavgeek  can you please check this document. Some lines seem to be incomplete